interfaces/ldconfig/spec.go

```diff
@@ -102,6 +108,9 @@ func (spec *Specification) AddConnectedSlot(iface interfaces.Interface, plug *in
 
 // AddPermanentPlug records ldconfig-specific side-effects of having a plug.
 func (spec *Specification) AddPermanentPlug(iface interfaces.Interface, plug *snap.PlugInfo) error {
+	// Keep track of interfaces using this backend on the consumer side
```

Comment:
shouldn't we do this only if the interface has actually LdconfigPermanentPlug or LdconfigConnectedPlug implemented? otherwise I think this method is called for all plugs

Reply:
I check now for LdconfigConnectedPlug, I want to avoid the interface to implement an empty LdconfigPermanentPlug.

Reply:
I thought you mentioned you need to run things even if the plug is not connected? as I said you can check for either definer in AddPermantPlug

Reply:
Yes, otherwise we do not know if we need to remove configuration if all connections are gone.

Reply:
Latest commit checks the definer in AddPermantPlug